tests/pipeline_operations_test.py

```diff
@@ -133,6 +153,52 @@ def assert_laziness(operator, *args):
         assert_laziness(self.ops.values)
         assert_laziness(self.ops.count_per_element)
 
+    def test_local_sample_fixed_per_key_requires_no_discarding(self):
```

Comment:
please add 
assert_laziness(self.ops.sample_fixed_per_key) 
assert_laziness(self.ops.flat_map) 

in test_laziness a few lines above

Reply:
thanks!